Description: Correct type mismatch in gperf generated file
 The hconfig.c file, generated from gperf and distributed by
 upstream, is not properly patched yet and does not build
 correctly. This patch avoid a type mismatch between the
 prototype and the function definition.
Author: Philippe Thierry <phil@reseau-libre.net>
--- a/libs/parser/hconf.c
+++ b/libs/parser/hconf.c
@@ -172,7 +172,7 @@
 #endif
 #endif
 const struct kconf_id *
-kconf_id_lookup (register const char *str, register unsigned int len)
+kconf_id_lookup (register const char *str, register GPERF_LEN_TYPE len)
 {
   enum
     {
